cmd/tether/attach.go

```diff
@@ -464,6 +483,13 @@ func (t *attachServerSSH) globalMux(reqchan <-chan *ssh.Request) {
 			}
 			msg := msgs.ContainersMsg{IDs: keys}
 			payload = msg.Marshal()
+
+			// unblock ^ (above)
+			pendingFn = func() {
+				once.Do(func() {
+					close(t.askedAndAnswered)
```

Comment:
@caglar10ur this will work given our current logic, but is there an inherent reason why? If a channel request is made with a legitimate ID, even without asking, is that not the trigger for launch?